README.md

```diff
@@ -199,16 +170,18 @@ balanceRequest.run(new ResultCallback<Balance>() {
 });
 ```
 
-### Transfering KIN to another account
-To transfer KIN to another account, you need the public address of the account you want 
-to transfer the KIN to. 
+## Transactions
+
+### Transferring KIN to another account
+
+To transfer KIN to another account, you need the public address of the account you want to transfer the KIN to.
+
+The following code will transfer 20 KIN to account "GDIRGGTBE3H4CUIHNIFZGUECGFQ5MBGIZTPWGUHPIEVOOHFHSCAGMEHO".
```

Comment:
Mention this is the recipient account public address

Reply:
:+1: